main/src/ca/uwaterloo/flix/language/phase/HtmlDocumentor.scala

```diff
@@ -174,11 +176,20 @@ object HtmlDocumentor {
     * but with all items that shouldn't appear in the documentation removed.
     */
   private def filterItems(mods: Map[Symbol.ModuleSym, Module]): Map[Symbol.ModuleSym, Module] = mods.map {
-    case (sym, Module(_, parent, uses, submodules, classes, enums, effects, typeAliases, defs)) =>
+    case (sym, Module(_, parent, uses, mainItem, submodules, classes, enums, effects, typeAliases, defs)) =>
       sym -> Module(
         sym,
         parent,
         uses,
+        mainItem match {
```

Comment:
is mainItem more like what we call the Prelude or RootModule?

Reply:
I'm not sure I understand. 
`mainItem` is the enum/class that's displayed at the top of the given module.

Reply:
Ah okay. Its just that the word "main" to me means the "main function".

Reply:
I see. Do you propose an alternative?